### ISSUE TITLE
Cross-browser Blockstack state being reset

### ISSUE THREAD

**Issue by jay-ithiel** [stx-labs/stacks.js#237]
Not sure if this is a known issue, but I noticed some discrepancies between different browsers on Blockstack

Safari doesn't have access to my Blockstack User Data:
<img width="612" alt="screen shot 2017-07-29 at 11 36 58 am" src="https://user-images.githubusercontent.com/20843566/28747386-5dab925a-7452-11e7-8a79-680711e0adb2.png">

But Chrome does:
<img width="601" alt="screen shot 2017-07-29 at 11 37 16 am" src="https://user-images.githubusercontent.com/20843566/28747403-9b1fbf58-7452-11e7-89eb-b5750d35c0cc.png">

On Chrome, the blogs are fetched:
<img width="1278" alt="screen shot 2017-07-29 at 11 24 40 am" src="https://user-images.githubusercontent.com/20843566/28747339-32c22078-7451-11e7-9bbb-8d9b63173300.png">

But on Safari, the blogs are not fetched: 
<img width="1280" alt="screen shot 2017-07-29 at 11 24 54 am" src="https://user-images.githubusercontent.com/20843566/28747338-32c1f03a-7451-11e7-8274-91761727d79a.png">

Posting a blog on Safari will not appear on Chrome.

New Blog post made on Safari:
<img width="666" alt="screen shot 2017-07-29 at 11 41 39 am" src="https://user-images.githubusercontent.com/20843566/28747422-fc393c06-7452-11e7-9d48-9d0b7c28dceb.png">
Doesn't appear on Chrome:
<img width="557" alt="screen shot 2017-07-29 at 11 42 04 am" src="https://user-images.githubusercontent.com/20843566/28747423-fc3bd934-7452-11e7-9d34-4e28811153bc.png">

Chrome is set to the default Blockstack Browser on my computer, so I'm assuming that probably has something to do with Safari not being able to access my Blockstack user/storage info. However, it might be confusing if users do something on a Blockstack App using Safari, but don't see the changes when accessing the app using a different browser.

**Comment by jay-ithiel** [comment]
I just tested this on Firefox, and the blog posts I made on Safari show up on Firefox and vice versa. It's just Chrome that doesn't share data with Safari & Firefox. 
Firefox also doesn't have access to my Blockstack account data. I'm assuming this is because Chrome is set to my default Blockstack browser.

Blogs written on Safari are shown on Firefox
<img width="1280" alt="screen shot 2017-07-29 at 12 41 03 pm" src="https://user-images.githubusercontent.com/20843566/28747767-94a6026e-745b-11e7-98b7-5c8ea89bd41a.png">
<img width="1280" alt="screen shot 2017-07-29 at 12 41 15 pm" src="https://user-images.githubusercontent.com/20843566/28747768-94a7e03e-745b-11e7-8345-de96890736e0.png">
